main.py

```diff
@@ -11,13 +11,24 @@
 @app.route('/v1/todos')
 @app.route('/v1/todos/')
 @app.route('/v1/todos/<status>')
-def list_all(status=ALL):
+def list_all(status=ALL) -> object:
     todos_list = todos_repo.list_all(status)
     return jsonify({
-            'items': [
-                x.to_dict() for x in todos_list
-            ],
-            'details': {
-                'activeToDos': sum(map(lambda x: x.has_status(Status.ACTIVE.value), todos_list))
-            }
-        })
+        'items': [
+            x.to_dict() for x in todos_list
+        ],
+        'details': {
+            'activeToDos': sum(map(lambda x: x.has_status(Status.ACTIVE.value), todos_list))
+        }
+    })
+
+
+@app.route('/v1/todos', methods=['POST'])
+@app.route('/v1/todos/', methods=['POST'])
+def create():
+    to_do = request.get_json()
+    if not to_do.get('name') or len(to_do.keys()) != 1:
+        abort(400)
```

Comment:
This enough to know that the client didn't make a good request, but it is not enough to know what is the reason